src/main/java/io/ark/core/model/Account.java

```diff
@@ -1,68 +1,49 @@
 package io.ark.core.model;
 
-import static io.ark.core.util.Constants.ARKTOSHI;
-
-import java.text.MessageFormat;
 import java.util.ArrayList;
 import java.util.List;
 
 import org.bitcoinj.core.ECKey;
 
-import io.ark.core.network.NetworkInfo;
-import io.ark.core.requests.AccountManager;
-import io.ark.core.requests.TransactionManager;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
+@Builder
+@AllArgsConstructor
 @NoArgsConstructor
 public class Account {
 	private String address;
 	private long unconfirmedBalance;
 	private long balance;
 	private String publicKey;
-	private int unconfirmedSignature;
-	private int secondSignature;
-	private String secondPublicKey;
-	private List<String> multisignatures;
-	private List<String> u_multisignatures;
-	
-	private NetworkInfo info;
-	private ECKey keyPair;
+  private int unconfirmedSignature;
```

Comment:
Sorry for this massacre. I'm going to add some editor configs probably in a contribution area along with checkstyle rules so this nastiness doesn't happen again. (i'll fix this in the next pr no need for a tiny commit for this)